test.py

```diff
@@ -169,6 +169,10 @@ def test_subscriber_names(self):
     def test_subscribe(self):
         self.qiwis.subscribe("app1", "ch3")
         self.assertIn("app1", self.qiwis._subscribers["ch3"])
```

Comment:
How about asserting that `"app1"` was not in `_subscribers["ch3"]` before subscribing?

Reply:
Good! I will apply it.